server/src/configs/default.json

```diff
@@ -30,10 +30,11 @@
     "title": "ReactMap",
     "headerTitle": "ReactMap",
     "nestHemisphere": "north",
-    "enableFeedbackLink": false,
+    "enableFeedback": false,
```

Comment:
I would remove this change from this PR since it's coming in your local auth PR anyway, just to reduce conflicts. After that, I can merge this in.